### PR TITLE
updating Readme

### PR DESCRIPTION
**Pull request by programad**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,26 +19,29 @@ This component provides a collection of icons for use in your **React** web and
...
 
-- Next.js 14
+## Packages in this repository
 
-## Preparing the environment
+- [core](https://github.com/rocketclimb/rocketicons/blob/main/packages/core) - The core of the icon package generation.
+- [gen](https://github.com/rocketclimb/rocketicons/blob/main/packages/gen) - The icon generator itself.
+- [icons](https://github.com/rocketclimb/rocketicons/blob/main/packages/icons) - The main package, generated by gen.
+- [ignition](https://github.com/rocketclimb/rocketicons/blob/main/packages/ignition) - Our website.
+- [sh](https://github.com/rocketclimb/rocketicons/blob/main/packages/sh) - Shell utilities to help things to work better.
```
Comment (amorimjj) [inline README.md:32]: We're going to remove it from this repository

File: README.md [hunk 2]
```diff
@@ -16,26 +19,29 @@ This component provides a collection of icons for use in your **React** web and
...
 
-## Technical specifications
+We also got excited about [@expo](https://github.com/expo) and the possibility of creating multiplatform apps using a single codebase, sharing as much code as we can between multiple platforms.
 
-- Next.js 14
+## Packages in this repository
 
-## Preparing the environment
+- [core](https://github.com/rocketclimb/rocketicons/blob/main/packages/core) - The core of the icon package generation.
+- [gen](https://github.com/rocketclimb/rocketicons/blob/main/packages/gen) - The icon generator itself.
```
Comment (amorimjj) [inline README.md:29]: I think we should rename gen. Gen it the name the original repository used, maybe change this name to generator to disassociate from the original work can be better...

**Review by amorimjj — COMMENTED** [review]
I've dropped few comments

File: README.md (hunk 2 above)
Comment (programad) [reply README.md:29]: Agree

**Review by amorimjj — APPROVED** [review]
code 👍

**Pull request merged**